packages/diagnostic/src/index.ts

```diff
@@ -0,0 +1 @@
+export { diagnose } from './disgnostic';
```

Comment:
typo here with the filename. additionally these should/could match eg.
`export { diagnose } from './diagnose'`;